fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsRepository.java

```diff
@@ -179,15 +205,7 @@ private static void dropStatistics(Long dbId,
 
         Map<String, String> params = new HashMap<>();
         params.put("condition", predicate.toString());
-        StringSubstitutor stringSubstitutor = new StringSubstitutor(params);
-
-        try {
-            String statement = isHistogram ? stringSubstitutor.replace(DROP_TABLE_HISTOGRAM_TEMPLATE) :
-                    stringSubstitutor.replace(DROP_TABLE_STATISTICS_TEMPLATE);
-            StatisticsUtil.execUpdate(statement);
-        } catch (Exception e) {
-            LOG.warn("Drop statistics failed", e);
-        }
+        return params;
     }
 
     private static <T> void buildPredicate(String fieldName, Set<T> fieldValues, StringBuilder predicate) {
```

Comment:
generic type doesn't make any sense

Reply:
fieldValues could be Set<Long> tblIds, or Set<String> colNames... all converted in this method

Reply:
you're right, thx